jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -372,7 +372,7 @@ public void testBuild_complex()
     Assert.assertNotEquals(digest, id);
     Assert.assertEquals(output, new Command("docker", "run", "--rm", id).run());
 
-    JibRunHelper.assertSimpleCreationTimeIsAfter(beforeBuild, targetImage);
+    assertThat(JibRunHelper.getCreationTime(targetImage)).isGreaterThan(beforeBuild);
     assertThat(getWorkingDirectory(targetImage)).isEqualTo("/");
```

Comment:
let's also convert the other assert statements in this test to Truth.

Reply:
I've been doing that, but there are too many. I think I'll just do it on the best effort basis. Don't want to spend too much time on low priority tasks for now.

Reply:
Makes sense. Having a mix of Truth and Junit is usually okay (neither discouraged nor encouraged) - we can just migrate in a separate PR when possible.